x-pack/plugins/security/server/saved_objects/saved_objects_security_extension.ts

```diff
@@ -38,12 +40,100 @@ export class SavedObjectsSecurityExtension implements ISavedObjectsSecurityExten
   private readonly auditLogger: AuditLogger;
   private readonly errors: SavedObjectsClient['errors'];
   private readonly checkPrivilegesFunc: CheckSavedObjectsPrivileges;
+  private readonly actionMap: Map<
+    SecurityAction,
+    { authzAction: string | undefined; auditAction: AuditAction | undefined }
+  >;
 
   constructor({ actions, auditLogger, errors, checkPrivileges }: Params) {
     this.actions = actions;
     this.auditLogger = auditLogger;
     this.errors = errors;
     this.checkPrivilegesFunc = checkPrivileges;
+
+    // OPERATION                          ES AUTH ACTION          AUDIT ACTION
+    // -----------------------------------------------------------------------------------------
+    // Check Conflicts                    'bulk_create'           N/A
+    // Close PIT                          N/A                     AuditAction.CLOSE_POINT_IN_TIME
+    // Collect References                 'bulk_get'              AuditAction.COLLECT_MULTINAMESPACE_REFERENCES
+    // Collect Refs For Updating Spaces   'share_to_space'        AuditAction.COLLECT_MULTINAMESPACE_REFERENCES
+    // Create                             'create'                AuditAction.CREATE
+    // Bulk Create                        'bulk_create'           AuditAction.CREATE
+    // Delete                             'delete'                AuditAction.DELETE
+    // Bulk Delete                        'bulk_delete'           AuditAction.DELETE
+    // Find                               'find'                  AuditAction.FIND
+    // Get                                'get'                   AuditAction.GET
+    // Bulk Get                           'bulk_get'              AuditAction.GET
+    // Internal Bulk Resolve              'bulk_get'              AuditAction.RESOLVE
+    // Open PIT                           'open_point_in_time'    AuditAction.OPEN_POINT_IN_TIME
+    // Remove References                  'delete'                AuditAction.REMOVE_REFERENCES
+    // Update                             'update'                AuditAction.UPDATE
+    // Bulk Update                        'bulk_update'           AuditAction.UPDATE
+    // Update Objects Spaces              'share_to_space'        AuditAction.UPDATE_OBJECTS_SPACES
+    this.actionMap = new Map([
```

Comment:
Because the "PerformAuthorization" method of the extension will likely be dissolved we can also eliminate this action map as well, though I kind of appreciate the organization of it over raw strings as actions. Curious what others think about this.

Reply:
Yeah, I'd probably keep this map as well, just to make it easier to understand the relationship between authz and audit actions.